docs/marionette.object.md

```diff
@@ -78,28 +74,14 @@ radioRequests: {
 
 [Live example](https://jsfiddle.net/marionettejs/3seo87o1/)
 
-means that the object will listen for the `doFoo` request on the `app` channel, and run the 'executeFoo' method.  When using Radio Requests with Objects, the same rules and restrictions that normal Radio use implies also apply here: a single handler can be associated with a request, either through manual use of the reply functions, or through the Object API.
-
-## mergeOptions
-Merge keys from the `options` object directly onto the instance. This is the preferred way to access options
-passed into the Object.
-
-More information at [mergeOptions](./marionette.functions.md#marionettemergeoptions)
-
-## getOption
-Retrieve an object's attribute either directly from the object, or from the object's this.options, with this.options taking precedence.
-
-More information [getOption](./marionette.functions.md#marionettegetoption).
-
-## bindEntityEvents
-Helps bind a backbone "entity" to methods on a target object. More information [bindEntityEvents](./marionette.functions.md#marionettebindentityevents).
+In this example the object will listen for the `some:request` request on the `myChannel` channel, and run the 'requestHandler' method.  When using Radio Requests with Objects, the same rules and restrictions that normal Radio use implies also apply here: a single handler can be associated with a request, either through manual use of the reply functions, or through the Object API.
 
 ## Destroying A Object
 
 Objects have a `destroy` method that unbind the events that are directly attached to the
 instance.
 
-Invoking the `destroy` method will trigger a "before:destroy" event and corresponding
+Invoking the `destroy` method will trigger a `before:destroy` event and corresponding
 `onBeforeDestroy` method call. These calls will be passed any arguments `destroy`
```

Comment:
I meant using the `**Note**` to tell readers that `destroy` will pass its arguments into its events and this is different to other event handling stuff.